source/mongodb/crud-and-aggregation-apis.txt

```diff
@@ -529,74 +529,66 @@ using the following commands:
 Aggregation Pipeline Stage Availability
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-App Services does not support the following :manual:`aggregation pipeline stages
-</reference/operator/aggregation-pipeline/>` when you :ref:`run an aggregation
-pipeline <mongodb-aggregate>` in the context of an
-:ref:`application user <user-accounts>`. All aggregation pipeline stages are
-available to the system user except for ``$indexStats``.
+All :manual:`aggregation pipeline stages </reference/operator/aggregation-pipeline/>` 
+stages are available to the *system user* except for ``$indexStats``. However, 
+App Services does not support the following aggregation pipeline stages
+when you :ref:`run an aggregation pipeline <mongodb-aggregate>` in the context 
+of an :ref:`application user <user-accounts>`. 
 
 .. list-table::
    :header-rows: 1
-   :widths: 25 10 10
-
-   * - Stage
-     - User Context
-     - System Context
 
-   * - :manual:`$collStats </reference/operator/aggregation/collStats/>`
-     - No
-     - Yes
+   * - Operation
+     - Notes
+     
+   * - :manual:`$collStats </reference/operator/aggregation/collStats/>` 
+     - 
 
    * - :manual:`$currentOp </reference/operator/aggregation/currentOp/>`
-     - No
-     - Yes
-
+     - 
+          
    * - :manual:`$facet </reference/operator/aggregation/facet/>`
-     - No
-     - Yes
-
+     - 
+          
    * - :manual:`$geoNear </reference/operator/aggregation/geoNear/>`
-     - No
-     - Yes
-
+     - 
+          
    * - :manual:`$graphLookup </reference/operator/aggregation/graphLookup/>`
-     - No
-     - Yes
-
+     - 
+          
    * - :manual:`$indexStats </reference/operator/aggregation/indexStats/>`
-     - No
-     - No
-
+     - 
+          
    * - :manual:`$lookup </reference/operator/aggregation/lookup/>`
-     - Yes
-     - Yes
-
+     - 
+          
    * - :manual:`$merge </reference/operator/aggregation/merge/>`
-     - No
-     - Yes
-
+     - 
+          
    * - :manual:`$out </reference/operator/aggregation/out/>`
-     - No
-     - Yes
-
+     - |
+       | By default, ``$merge`` and ``$out`` route the entire aggregation operation to
+       | the cluster's primary node. If you want to force these stages to respect your
+       | configured cluster read preference, set the ``enforceReadPref`` option to
+       | ``true``.
+     
    * - :atlas:`$search </reference/atlas-search/query-syntax/>`
-       
-       .. include:: /includes/note-atlas-search-rules.rst
-
-       .. include:: /includes/note-atlas-search-search_meta.rst
-     - Yes
-     - Yes
-
+     - |
+       | App Services performs ``$search`` operations as a system user and
+       | enforces field-level rules on the returned search results. This means 
+       | that a user may search on a field for which they do not have read access. 
+       | In this case, the search is based on the specified field but no returned 
+       | documents include the field. 
+     
```

Comment:
There's a build error: 
>  WARNING(mongodb/crud-and-aggregation-apis.txt:583ish): Inline interpreted text or phrase reference start-string without end-string.

I don't remember if rST line bock syntax requires uninterrupted lines. If you add another pipe here, does the error go away?
```suggestion
     |
```